src/context/display/statistics.ml

```diff
@@ -70,21 +70,24 @@ let collect_statistics ctx pfilter with_expressions =
 		) c.cl_overrides
 	in
 	let collect_implementations c =
-		List.iter (fun cf ->
-			let rec loop c =
-				begin try
-					let cf' = PMap.find cf.cf_name c.cl_fields in
-					add_relation cf.cf_name_pos (Implemented,cf'.cf_name_pos)
-				with Not_found ->
-					()
-				end;
-				List.iter loop c.cl_descendants
-			in
-			List.iter loop c.cl_descendants
-		) c.cl_ordered_fields;
+		let memo = Hashtbl.create 0 in
```

Comment:
What would be a good initial size here?

Reply:
I've experimented with that a while back and concluded that `0` was generally the best value.